MediaBrowser.Providers/Lyric/ILyricProvider.cs

```diff
@@ -20,17 +21,10 @@ public interface ILyricProvider
     /// <value>The priority.</value>
     ResolverPriority Priority { get; }
 
-    /// <summary>
-    /// Checks if an item has lyrics available.
-    /// </summary>
-    /// <param name="item">The media item.</param>
-    /// <returns>Whether lyrics where found or not.</returns>
-    bool HasLyrics(BaseItem item);
-
     /// <summary>
     /// Gets the lyrics.
     /// </summary>
-    /// <param name="item">The media item.</param>
+    /// <param name="path">The lyric path.</param>
     /// <returns>A task representing found lyrics.</returns>
-    Task<LyricFile?> GetLyrics(BaseItem item);
+    Task<LyricFile?> GetLyrics(string path);
```

Comment:
This will break the plugin support I added in #9920. With BaseItem a plugin can find lyrics based on metadata (artist/track name) from various sources, including the file system but also online services etc.

Reply:
Good call, I've reverted that change and now plugins can specify whether the item has lyrics and can also provide them